contributors/devel/sig-release/release.md

```diff
@@ -194,57 +194,42 @@ Each release is part of a broader Kubernetes lifecycle:
+- SIG leadership (SIG Chairs / Technical Leads) from all SIGs
+- SIG milestone maintainers from all SIGs (in addition to SIG leadership)
+- Special code reviewers, as selected by SIG Release
 
-Where additional action is required, an attempt at human to human escalation
-will be made by the Release Team through the following channels:
+Milestone maintainers also have write access to [kubernetes/enhancements][k/enhancements], which allows them to keep enhancement tracking issue descriptions up-to-date.
 
-- Comment in GitHub mentioning the SIG team and SIG members as appropriate for
-  the issue type
-- Emailing the SIG mailing list
-  - bootstrapped with group email addresses from the
-    [community sig list][sig-list]
-  - optionally also directly addressing SIG leadership or other SIG members
-- Messaging the SIG's Slack channel
-  - bootstrapped with the slackchannel and SIG leadership from the
-    [community sig list][sig-list]
-  - optionally directly "@" mentioning SIG leadership or others by handle
+To request membership to `kubernetes-milestone-maintainers` or update the team:
+- File a PR to [kubernetes/org][k/org] making changes to the `kubernetes-milestone-maintainers` team config [here](https://git.k8s.io/org/config/kubernetes/sig-release/teams.yaml). In your PR, include the reason you're requesting access in the description, as well as a comment next to your username in the team config
+  e.g.,
+  ```
+  - justaugustus # Azure / PM / Release
+  - spiffxp # 1.14 RT Lead / Testing
+  ```
+- Assign an approver from the group you'll be maintaining issues/PRs for
+  i.e.,
+  - for SIGs: Request approval from a SIG Chair or Technical Lead
+  - for Release Team: Request approval from the current Release Team Lead or SIG Release Chairs
+  - for Special code reviewers: Request approval from SIG Release Chairs
 
-## Adding An Item To The Milestone
+- Once your request is approved by the appropriate group, you can now assign to the PR to a SIG Release Chair for final approval.
 
-### Milestone Maintainers
+The responsibles for the `kubernetes-milestone-maintainers` GitHub team are defined as follows:
+- [SIG Release Chairs](https://github.com/kubernetes/sig-release/blob/master/release-managers.md#sig-release-leads)
+- Current [Release Team Lead](https://github.com/kubernetes/sig-release/tree/master/releases)
 
-The members of the [`milestone-maintainers`](https://github.com/orgs/kubernetes/teams/milestone-maintainers/members)
-GitHub team are entrusted with the responsibility of specifying the release
-milestone on GitHub artifacts.
+They can, with justification, add or remove members at any time during release cycles of the GitHub team. Members are expected to actively [triage](/contributors/guide/issue-triage.md) issues and PRs to retain membership in `kubernetes-milestone maintainers`. Members not fulfilling the duties of this role should be removed.
```

Comment:
You'd want to rearrange a bit to say "add or remove members of the GitHub team at any time" :)